com.toxicstoxm.LEDSuite.yml

```diff
@@ -0,0 +1,37 @@
+id: com.toxicstoxm.LEDSuite
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+
+command: LEDSuite
+
+modules:
+
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+      - mkdir -p /app/bin
+      - ln -s /app/jre/bin/java /app/bin/java
+
+  - name: LEDSuite
+    buildsystem: meson
+    config-opts:
+      - -Djavadir=/usr/lib/sdk/openjdk
+
+    sources:
+      - type: git
+        url: https://github.com/ToxicStoxm/LEDSuite.git
+        tag: v1.0.0-rc3
```

Comment:
please add a commit as well. Looks like you retagged and retagged is bad. this will protect against it.